docs/resources/Guild.md

```diff
@@ -730,6 +730,8 @@ Create a new [channel](#DOCS_RESOURCES_CHANNEL/channel-object) object for the gu
 | rtc_region                    | string                                                                         | channel [voice region](#DOCS_RESOURCES_VOICE/voice-region-object) id of the voice or stage channel, automatic when set to null                                                  |
 | video_quality_mode            | integer                                                                        | the camera [video quality mode](#DOCS_RESOURCES_CHANNEL/channel-object-video-quality-modes) of the voice channel                                                                |
 | default_auto_archive_duration | integer                                                                        | the default duration that the clients use (not the API) for newly created threads in the channel, in minutes, to automatically archive the thread after recent activity         |
+| default_reaction_emoji        | [default reaction](#DOCS_RESOURCES_CHANNEL/default-reaction-object) object     | the emoji to show in the add reaction button on a thread in a `GUILD_FORUM` channel                                                                                             |
```

Comment:
```suggestion
| default_reaction_emoji?       | [default reaction](#DOCS_RESOURCES_CHANNEL/default-reaction-object) object     | emoji to show in the add reaction button on a thread in a `GUILD_FORUM` channel                                                                                             |
```

Reply:
i know "the" is common in this table, but trying to move away from it

Reply:
The table has a note saying that all parameters are optional besides name, and none of the others are marked as optional, so I just repeated this pattern:

> All parameters to this endpoint are optional and nullable excluding `name`

Reply:
ooo oops sorry didn't see that. i don't like that being present, but not on you to fix that in this PR.